src/main/scala/zio/kafka/admin/AdminClient.scala

```diff
@@ -296,15 +296,19 @@ object AdminClient {
     override def describeClusterNodes(options: Option[DescribeClusterOptions] = None): Task[List[Node]] =
       fromKafkaFuture(
         describeCluster(options).map(_.nodes())
-      ).map(_.asScala.toList.map(Node.apply))
+      ).map(_.asScala.toList.flatMap(Node.apply))
```

Comment:
@trobert Would you fail or silently flatten here?

Reply:
failling is better, but I can go with flatten if you find it makes the code too noisy

Reply:
Ok I change it with  a fail